src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -4,7 +4,7 @@ language_settings:
   java:
     package_name: com.google.gcloud.pubsub.spi
   python:
-    package_name: library
+    package_name: google.example.library.v1
```

Comment:
Ah, there wasn't much thought put into this change, I just wanted the test code to look a little more realistic because previously the package strings in the samples were unrealistically short.

It is probably more accurate to change this to `google.cloud.example.library.v1` given our current practice in reality.